shopsafe-frontend/src/app/components/check-in-modal/check-in-modal.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
could you add a comment on the top of this file for what it does?

Reply:
Added comments in latest commit!